flocker/node/agents/gce.py

```diff
@@ -172,13 +173,19 @@ def wait_for_operation(compute, operation, timeout_steps):
     """
     poller = _create_poller(operation)
 
-    def finished_operation_result():
-        latest_operation = poller.poll(compute)
-        if latest_operation['status'] == 'DONE':
-            return latest_operation
-        return None
+    with start_action(
+            action_type=u"flocker:node:agents:gce:wait_for_operation",
+            operation=operation
+    ) as action:
```

Comment:
I would suggest going for 4 space indents for these context manager arguments.  I know it looks better with 8 but, searching for `with start_action` in the rest of the codebase shows that the other folks elected to go with 4 spaces.